### PR TITLE
Factor out `wrapInElement` calls from tests

### PR DESCRIPTION
**Pull request by bkrmendy**
## Description
This PR factors out calls to the `wrapInElement` factory function from the browser tests, so that they don't assume that this exact action is used for wrapping (since the action will be replaced with a commands/strategies-based implementation by https://github.com/concrete-utopia/utopia/pull/4435).

## Details
- `wrap in div works inside a conditional on an expression` was x-ed, because it's an extreme edge case of wrapping, and fixing it would have bloated the PR

### CODE REVIEW

File: editor/src/utils/utils.test-utils.tsx [hunk 1]
```diff
@@ -575,3 +575,15 @@ export function boundingClientRectToCanvasRectangle(
...
 }
+
+export async function searchInFloatingMenu(editor: EditorRenderResult, query: string) {
+  const floatingMenu = editor.renderedDOM.getByTestId(CanvasToolbarSearchTestID)
+  const searchBox = queryByAttribute('type', floatingMenu, 'text')!
+
+  await act(() => {
+    fireEvent.focus(searchBox)
+    fireEvent.change(searchBox, { target: { value: query } })
+    fireEvent.blur(searchBox)
```
Comment (seanparsons) [inline editor/src/utils/utils.test-utils.tsx:586]: I'm not sure if it makes any difference here, but I'd normally expect `blur` to trigger after `keydown`.

**Review by seanparsons — APPROVED** [review]
